bazel/FuzzTargetTestWrapper.java

```diff
@@ -15,32 +15,47 @@
 import com.google.devtools.build.runfiles.Runfiles;
 import java.io.File;
 import java.io.IOException;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
+import javax.tools.JavaCompiler;
+import javax.tools.JavaCompiler.CompilationTask;
+import javax.tools.JavaFileObject;
+import javax.tools.StandardJavaFileManager;
+import javax.tools.ToolProvider;
 
 public class FuzzTargetTestWrapper {
   public static void main(String[] args) {
+    Runfiles runfiles;
     String driverActualPath;
+    String apiActualPath;
     String jarActualPath;
-    Runfiles runfiles;
+    boolean verifyCrashInput;
+    boolean verifyCrashReproducer;
+    boolean executeCrashReproducer;
     try {
       runfiles = Runfiles.create();
       driverActualPath = runfiles.rlocation(rlocationPath(args[0]));
-      jarActualPath = runfiles.rlocation(rlocationPath(args[1]));
+      apiActualPath = runfiles.rlocation(rlocationPath(args[1]));
+      jarActualPath = runfiles.rlocation(rlocationPath(args[2]));
+      verifyCrashInput = Boolean.parseBoolean(args[3]);
+      verifyCrashReproducer = Boolean.parseBoolean(args[4]);
+      executeCrashReproducer = Boolean.parseBoolean(args[5]);
     } catch (IOException | ArrayIndexOutOfBoundsException e) {
       e.printStackTrace();
       System.exit(1);
       return;
     }
 
-    ProcessBuilder processBuilder = new ProcessBuilder();
-    Map<String, String> environment = processBuilder.environment();
-    // Ensure that Jazzer can find its runfiles.
-    environment.putAll(runfiles.getEnvVars());
```

Comment:
We should set these env vars in the ProcessBuilder created below.

Reply:
The env vars didn't look Jazzer specific and all tests passed without. 
I can revert this but wonder which ones Jazzer needs.

Reply:
They are not Jazzer-specific, but encode information about how Jazzer can find it's data dependencies provided by Bazel when invoked by the test wrapper. This is always easy to do from the top-level target executed by Bazel, but subprocesses generally have to be passed these variable to find their runfiles in all scenarios (that is, even when executed with bazel run on Windows). If the variables are not passed, runfile discovery will be non-deterministic on Windows.

Reply:
Done. Thx for the explanation.